CIP-namesSurroundingFeatures/feature-table.md

```diff
@@ -0,0 +1,12 @@
+# Cardano Features
+
+| Date    | Phase   | Era     | Slot Number | Epoch Number | Protocol Version | Ledger Protocol | Consensus Mechanism | Notes              |
```

Comment:
Might be worth mentioning:
- Protocol version is not the same as [Network Protocol Version](https://github.com/input-output-hk/ouroboros-network/blob/master/ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Ledger/NetworkProtocolVersion.hs), which is at higher index than Protocol version referred by Ledger)? Not sure if it's something that can be mapped in this table, probably not because it's from networking layer between nodes?
- Perhaps a condition with link at the bottom to clarify about Protocol Versions in block header signify not the Protocol Version itself, but the maximum supported ledger-protocol-version for node?

Reply:
* indeed, the "protocol version" is separate from the node-to-node and node-to-client versions. I'm happy to make a note of this, but I'd need help lining the node-to-x versions with this table (and indeed, it may not even line up at all).
* will do!

Reply:
@rdlrt I added a commit to address these two points.